tt1-blocks/assets/css/blocks.css

```diff
@@ -179,23 +144,7 @@ hr.is-style-twentytwentyone-separator-thick,
 	border-bottom-width: 3px;
 }
 
-.wp-block-separator.is-style-dots {
-	border-bottom: none;
-}
-
-.wp-block-separator.is-style-dots > hr {
-	display: none;
-}
-
 /* In Twenty Twenty-One, separators are 100% wide. This replicates that approach */
 .wp-block-separator:not(.is-style-wide):not(.is-style-dots) {
 	width: 100%;
```

Comment:
> Active Installations: 100+
I think breaking some backward compatibility is OK because the users still gain more features that did not exist when the theme was released....